tests/integration/test_ar7_ft.py

```diff
@@ -0,0 +1,79 @@
+import platform
+
+import pandas as pd
+import pytest
+
+from cmip_ref.config import default_metric_providers
+from cmip_ref.database import Database
+from cmip_ref.models import MetricExecution
+
+
+def create_execution_dataframe(executions):
+    data = []
+
+    for execution in executions:
+        assert len(execution.results) == 1
+        result = execution.results[0]
+
+        data.append(
+            {
+                "metric": execution.metric.slug,
+                "provider": execution.metric.provider.slug,
+                "execution_id": execution.id,
+                "result_id": result.id,
+                "execution_key": execution.key,
+                "successful": result.successful,
+            }
+        )
+
+    return pd.DataFrame(data)
+
+
+@pytest.fixture
+def config_ar7_ft(config):
+    """
+    Overwrite the default test config to use the metric providers for AR7 FT
+    """
+    # Force the default metric providers
+    config.metric_providers = default_metric_providers()
+
+    # Write the config to disk so it is used by the CLI
+    # This overwrites the default config
+    config.save()
+
+    return config
+
+
+@pytest.mark.slow
+def test_solve_ar7_ft(
+    sample_data_dir,
+    config_ar7_ft,
+    invoke_cli,
+    monkeypatch,
+):
+    # Arm-based MacOS users will need to set the environment variable `MAMBA_PLATFORM=osx-64`
+    if platform.system() == "Darwin" and platform.machine() == "arm64":
+        monkeypatch.setenv("MAMBA_PLATFORM", "osx-64")
+
+    assert len(config_ar7_ft.metric_providers) == 3
+
+    db = Database.from_config(config_ar7_ft)
+
+    # Ingest the sample data
+    invoke_cli(["datasets", "ingest", "--source-type", "cmip6", str(sample_data_dir / "CMIP6")])
+    invoke_cli(["datasets", "ingest", "--source-type", "obs4mips", str(sample_data_dir / "obs4MIPs")])
+
+    # Solve
+    # This will also create conda environments for the metric providers
+    invoke_cli(["--verbose", "solve", "--timeout", f"{60 * 60}"])
+
+    executions = db.session.query(MetricExecution).all()
+    df = create_execution_dataframe(executions)
+    print(df)
+
+    # Check that all 3 metric providers have been used
+    # TODO: Update once the PMP metrics are solving
```

Comment:
@lee1043 FYI the integration test doesn't currently run any PMP metrics due to the missing obs4MIPs data. Do you have temporary URLs for these obs data so that we can add them to the sample data?

Reply:
obs4MIPs-complying reference datasets for PMP's modes of variability metrics are available via the following links:

- HadISST: https://pcmdiweb.llnl.gov/pss/pmpdata/obs4MIPs_PCMDI_monthly/MOHC/HadISST-1-1/mon/ts/gn/v20210727/ts_mon_HadISST-1-1_PCMDI_gn_187001-201907.nc

- NOAA-20CR: https://pcmdiweb.llnl.gov/pss/pmpdata/obs4MIPs_PCMDI_monthly/NOAA-ESRL-PSD/20CR/mon/psl/gn/v20210727/psl_mon_20CR_PCMDI_gn_187101-201212.nc

Could you add them to the workflow so PMP can be included for the test?